addons/metadata.demo.movies/demo.py

```diff
@@ -1,98 +1,143 @@
+                'plot': 'Plot yo', 
+                'tagline': 'Tag yo', 
+                'duration': 110, 
+                'mpaa': 'T', 
+                'trailer': '/home/akva/bunnies/unicorns.mkv', 
+                'genre': ['Action', 'Comedy'], 
+                'country': ['Norway', 'Sweden', 'China'], 
+                'credits': ['None', 'Want', 'To Admit It'], 
+                'director': ['spiff', 'spiff2'], 
+                'set': 'Spiffy creations', 
+                'setoverview': 'Horrors created by spiff', 
+                'studio': ['Studio1', 'Studio2'], 
+                'dateadded': '2016-01-01', 
+                'premiered': '2015-01-01', 
+                'showlink': ['Demo show 1']
+            }
+        )
                     #todo: missing actor thumb aspect
-        liz.setRating("imdb", 9, 100000, True )
-        liz.setRating("themoviedb", 8.9, 1000)
-        liz.setUniqueIDs({ 'imdb': 'tt8938399', 'tmdb' : '9837493' }, 'imdb')
-        liz.setCast([{'name': 'spiff', 'role': 'himself', 'thumbnail': '/home/akva/Pictures/fish.jpg', 'order': 2},
-                    {'name': 'monkey', 'role': 'orange', 'thumbnail': '/home/akva/Pictures/coffee.jpg', 'order': 1}])
-        liz.addAvailableArtwork('DefaultBackFanart.png', 'banner')
-        liz.addAvailableArtwork('/home/akva/Pictures/hawaii-shirt.png', 'poster')
-        liz.setAvailableFanart([{'image': 'DefaultBackFanart.png', 'preview': 'DefaultBackFanart.png'}, 
-                                {'image': '/home/akva/Pictures/hawaii-shirt.png', 'preview': '/home/akva/Pictures/hawaii-shirt.png'}])
-        xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=True, listitem=liz)
+        list_item.setRating("imdb", 9, 100000, True )
+        list_item.setRating("themoviedb", 8.9, 1000)
+        list_item.setUniqueIDs({ 'imdb': 'tt8938399', 'tmdb' : '9837493'}, 'imdb')
+        list_item.setCast(
+            [
+                {
+                    'name': 'spiff', 'role': 'himself', 
+                    'thumbnail': '/home/akva/Pictures/fish.jpg', 'order': 2
+                }, {
+                    'name': 'monkey', 'role': 'orange', 
+                    'thumbnail': '/home/akva/Pictures/coffee.jpg', 'order': 1
+                }
+            ]
+        )
+        list_item.addAvailableArtwork('DefaultBackFanart.png', 'banner')
+        list_item.addAvailableArtwork(
+            '/home/akva/Pictures/hawaii-shirt.png', 'poster'
+        )
+        list_item.setAvailableFanart(
+            [
+                {
+                    'image': 'DefaultBackFanart.png', 
+                    'preview': 'DefaultBackFanart.png'
+                }, {
+                    'image': '/home/akva/Pictures/hawaii-shirt.png', 
+                    'preview': '/home/akva/Pictures/hawaii-shirt.png'
+                }
+            ]
+        )
+        setResolvedUrl(handle = int(argv[1]), succeeded = True, listitem = list_item)
```

Comment:
I thikn we should also set the "path" for an item in order ot use `setResolvedUrl()`?

Reply:
That's only used in plugins/when you need to resolve the path to a playable one. This add-on is a scrapper :)

Reply:
Lol, hadn't seen that one. Was just reviewing another add-on that also missed that. Hence I thought it would be nice to add it  in a demo add-on. But not this one then 😄